python/tk_multi_workfiles/file_model.py

```diff
@@ -98,7 +98,7 @@ def data(self, role=QtCore.Qt.UserRole+1):
                 # just return the default implementation:
                 return QtGui.QStandardItem.data(self, role)
 
-        def setData(self, value, role=QtCore.Qt.UserRole+1):
+        def setData(self, value, role=QtCore.Qt.UserRole + 1):
```

Comment:
I realize this is a minor style fix, but I think we should move the role + 1 to a class variable instead of doing it here in the data methods.